firebase_admin/project_management.py

```diff
@@ -640,16 +607,17 @@ def _poll_app_creation(self, operation_name):
             wait_time_seconds = delay_factor * _ProjectManagementService.POLL_BASE_WAIT_TIME_SECONDS
             time.sleep(wait_time_seconds)
             path = '/v1/{0}'.format(operation_name)
-            poll_response = self._make_request('get', path, operation_name, 'Operation name')
+            poll_response, http_response = self._body_and_response('get', path)
             done = poll_response.get('done')
             if done:
                 response = poll_response.get('response')
                 if response:
                     return response
                 else:
-                    raise _PollingError(
-                        'Polling finished, but the operation terminated in an error.')
-        raise _PollingError('Polling deadline exceeded.')
+                    raise exceptions.UnknownError(
+                        'Polling finished, but the operation terminated in an error.',
+                        http_response=http_response)
```

Comment:
Also for your consideration: we can actually use `poll_response.get('error')` as the http_response argument -- if it is non-empty and non-null -- here instead of the full `http_response`.

Reply:
http_response should be the full `requests.Response` object returned by the `requests` library -- including headers, body and request information. So this is handled correctly.

Reply:
Thanks for the explanation. Sounds good.